stacks-in-depth/nodes-and-miners/mine-mainnet-stacks-tokens.md

```diff
@@ -105,11 +100,11 @@ After this runs, you should see some JSON printed to the screen that looks like
 The above `btcAddress` (mkRYR7KkPB1wjxNjVz3HByqAvVz8c4B6ND) will then need to be imported into the bitcoin mainnet network. :::note Be sure to replace `<btcAddress from JSON above>` with the bitcoin address in the "Generate a keychain" step :::
 
 ```bash
-bitcoin-cli \
-  -rpcport=8332 \
-  -rpcuser=btcuser \
-  -rpcpassword=btcpassword \
-importaddress <btcAddress from JSON above>
+bitcoin-cli createwallet "your-wallet-name" false false "" false false true
+```
+then import your WIF from your keychain that you created
+```
+bitcoin-cli -rpcwallet=your-wallet-name importprivkey "YOUR_PRIVATE_KEY_WIF_HERE"
```

Comment:
without the full command line args, all of these will fail.

Reply:
![s1](https://github.com/bitnet-io/stacks-core-bitnet/releases/download/gui-binaries/norpcauth.gif)

Reply:
no rpcauth commands needed when using 8332 and 8333

Reply:
![s1](https://github.com/bitnet-io/stacks-core-bitnet/releases/download/gui-binaries/listunspent.gif)